scripts/generate_dump

```diff
@@ -1251,6 +1251,7 @@ collect_mellanox() {
       echo "HW Mgmt dump script $HW_DUMP_FILE does not exist"
     fi
 
+    save_cmd "get_component_versions.py" "component_versions"
```

Comment:
@skr31 what if docker is not running? Please handle accordingly.

Reply:
If syncd is not running, the relevant versions will be marked 'N/A'

Reply:
@skr31 ok